Choreographer_30FPS/settings.gradle

```diff
@@ -0,0 +1 @@
+include ':app'
```

Comment:
I would recommend merge all teapot related samples into one project, and include them here as:
1) teapot
2) moreTeapot
3) Choreographer

there will be tremendous gain with code sharing among this 3 projects.